distributions/Jamulus.1

```diff
@@ -60,82 +60,82 @@ is also possible.
 .Pp
 One participant starts
 .Nm
-in server mode, ideally on a dedicated server (virtual) machine;
-all participants start the (graphical) client which transmits audio
-to the server, receiving back a mixed stream.
+in Server mode, ideally on a dedicated Server (virtual) machine;
+all participants start the (graphical) Client which transmits audio
+to the Server, receiving back a mixed stream.
```

Comment:
lowercase

Reply:
The "Server" here is the Jamulus Server instance, not the machine it's running on.